src/Message/RefundResponse.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+namespace MyOnlineStore\Omnipay\KlarnaCheckout\Message;
+
+use Omnipay\Common\Message\ResponseInterface;
+
+final class RefundResponse extends AbstractResponse implements ResponseInterface
+{
+    /**
+     * @inheritDoc
+     */
+    public function isSuccessful()
+    {
+        return true;
```

Comment:
It is decided to postpone this to a later stage where the requests will be re-implemented using the httpclient interface instead of the `kco_rest` package.